frontend/templates/cucumber/.github/workflows/browser.yml

```
@@ -0,0 +1,42 @@
+name: Run Browser Tests
+
+on:
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - ready_for_review
+      - synchronize
```

```suggestion
on: pull_request
```
This does the same thing - by default GHA triggers for all PR event types